src/app/crypto.py

```
@@ -1,31 +1,53 @@
-        "OAUTH_ENCRYPTION_KEY must be configured before using the crypto helpers"
+
+def _require_fernet() -> Fernet:
+    global _fernet
+
+    if _fernet is not None:
+        return _fernet
+
+    if not OAUTH_ENCRYPTION_KEY:
+        raise RuntimeError(
+            "OAUTH_ENCRYPTION_KEY must be configured before using the crypto helpers"
+        )
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)